### PR TITLE
Update CRD manifests

### PR DESCRIPTION
**Pull request by benashz**
Missed a `make manifest` do to this being a docs PR. Will update the GH workflow in another PR to catch this next time.

### CODE REVIEW

File: chart/crds/secrets.hashicorp.com_hcpauths.yaml [hunk 1]
```diff
@@ -68,11 +68,11 @@ spec:
                   reasons, only project-level service principals should ever be used.
                 properties:
                   secretRef:
-                    description: "SecretRef is the name of a Kubernetes secret in
-                      the consumer's (VDS/VSS/PKI/HCP) namespace which provides the
-                      HCP ServicePrincipal clientID, and clientSecret. \n The secret
-                      data must have the following structure { \"clientID\": \"clientID\",
-                      \"clientSecret\": \"clientSecret\", }"
+                    description: 'SecretRef is the name of a Kubernetes secret in
+                      the consumer''s (VDS/VSS/PKI/HCP) namespace which provides the
```
Comment (kschoche) [inline chart/crds/secrets.hashicorp.com_hcpauths.yaml:72]:
```quote
                      the consumer''s (VDS/VSS/PKI/HCP) namespace which provides the
```

I'm confused, does this work? I see two ticks here, wont that break the string, or remove the punctuation?
Same potential problem in the other file?

Comment (benashz) [reply chart/crds/secrets.hashicorp.com_hcpauths.yaml:72]: Not sure, this is generated from `make manifests`

Comment (benashz) [reply chart/crds/secrets.hashicorp.com_hcpauths.yaml:72]: The CRDs are always synced to the chart, as well.

Comment (benashz) [reply chart/crds/secrets.hashicorp.com_hcpauths.yaml:72]: Seems like it is just escaping the single quote:
```
$ kubectl -n vault-secrets-operator-system get crds hcpauths.secrets.hashicorp.com -o yaml|grep consumer
                      the consumer's (VDS/VSS/PKI/HCP) namespace which provides the
```

Comment (kschoche) [reply chart/crds/secrets.hashicorp.com_hcpauths.yaml:72]: Wild, alright then :D 👍🏽

**Review by kschoche — APPROVED** [review]

**Pull request merged**